### ISSUE TITLE
loading wheel always displayed. Full emoji list doesn't appear

### ISSUE THREAD

**Issue by slambert** [gdpelican/retort#72]
Just updated discourse and retort.

Let me know if there's more info I can provide.

Here's what I'm seeing:

![Screen Shot 2020-09-02 at 12 06 38 PM](https://user-images.githubusercontent.com/67271/92008237-eaa48200-ed14-11ea-8c01-464666e69a7d.png)

![Screen Shot 2020-09-02 at 12 06 48 PM](https://user-images.githubusercontent.com/67271/92008236-eaa48200-ed14-11ea-9b14-6b59a98758be.png)

**Comment by vknow360** [comment]
We were facing same issue since yesterday but today's update solved this.

**Comment by angusmcleod** [comment]
Yes, this is fixed: https://github.com/gdpelican/retort/commit/272fe4c643864a9a419248abc2fd2e81258d3478
